nodes/nomos-node-api/src/http/backend/axum.rs

```diff
@@ -87,6 +92,15 @@ where
     }
 }
 
+macro_rules! sugar {
```

Comment:
Why is it called `sugar`??? Will it return me a 🍬 ? 😂 
Please add a more straight forward name!

Reply:
Ah, ok, good name!